usr/lib/common/loadsave.c

```diff
@@ -978,10 +1044,9 @@ CK_RV load_masterkey_so_old(STDLL_TokData_t *tokdata)
     // this file gets created on C_InitToken so we can assume that it always
     // exists
     //
-    sprintf(fname, "%s/MK_SO", tokdata->data_store);
-    fp = fopen(fname, "r");
+    fp = open_token_data_store_path(fname, sizeof(fname), tokdata, "MK_SO",
+                                    "r");
```

Comment:
are you really sure you want to break this line with just an "r"); on the next ?
I would not hesitate to move this back to the prev line (and get some complain from any checker or whatever).

Reply:
If it is okay with the project, I am all for it.  Felt silly splitting the line there, but according to the project rules 

> the limit on the length of lines is 80 columns and this is a strongly preferred limit

Will change it anyway.

Reply:
For users like me who like to vertically split their editors multiple times, im still a fan of the 80 columns limit, but i dont mind a few characters more here and there ..
Regarding line breaks in function argmument lists, i like to break the line such that all arguments stay between the brakets i.e., the last line ist the longest e.g.,
```
fp = open_token_data_store_path(fname, sizeof(fname),
                                tokdata, "MK_SO", "r");
```
However, since its just about a few characters more here, im okay with it.